scipy/stats/tests/test_stats.py

```diff
@@ -4450,6 +4450,15 @@ def test_nist(self):
             assert_allclose(res[0], f, rtol=rtol,
                             err_msg='Failing testcase: %s' % test_case)
 
+    def test_constant_values(self):
```

Comment:
So when you put in the warning you'll need to capture that in the test otherwise the test will fail because of the warning.
Here is an example from `spearman`
https://github.com/scipy/scipy/blob/master/scipy/stats/tests/test_stats.py#L312-L319

however in this PR you can check/assert the values returned are as expected. In the PR that Warren wrote there were also numerical precision loss which was all part of a (major) overhaul of `pearson`.

Reply:
I thought we didn't come to common decision on warning in the discussion of #11669 , that's why I didn't add it here. In particular, I didn't investigate "numerical issue", which is for nearly constant input as I guess.